src/pipecat/services/anthropic/llm.py

```diff
@@ -120,21 +119,19 @@ def create_context_aggregator(
         self,
         context: OpenAILLMContext,
         *,
-        user_kwargs: Mapping[str, Any] = {},
-        assistant_kwargs: Mapping[str, Any] = {},
```

Comment:
In the changelog, we mention that these parameters are deprecated, but here we are removing them. 

Is that intentional ? I think we should fix either the changelog or here.

Reply:
Ooops, they are not deprecated. They are just removed. No one has ever used those anyways.

Reply:
Changelog updated.